bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/altair/AltairParams.java

```diff
@@ -244,24 +246,24 @@ static public GuideStarType getGuideStarType(String name, GuideStarType value) {
      */
     public enum Mode implements DisplayableSpType, SequenceableSpType {
```

Comment:
My take on REL-2323 is that there are new modes, I think this list should be expanded. Maybe we don't need to let the user select them on the UI and they are purely to be made available via AGS